composer.json

```diff
@@ -11,19 +11,19 @@
     }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.7|~3.1|~4.0",
-        "symfony/finder": "~2.7|~3.1|~4.0",
-        "symfony/console": "~2.7|~3.1|~4.0",
-        "symfony/intl": "~2.7|~3.1|~4.0",
-        "symfony/translation":  "~2.7|~3.1|~4.0",
-        "symfony/twig-bundle": "~2.7|~3.1|~4.0"
+        "symfony/framework-bundle": "4.4.*",
```

Comment:
Using this we are supporting only symfony 4.4. We need to put the pipe as before.

Reply:
I changed it to: ~4.4|~5.0 

The changes I made are breaking for Symfony 4.3 and before.